dbt_sugar/core/documentation/documentation_helper.py

```diff
@@ -0,0 +1,159 @@
+"""Contains documentation related utils."""
```

Comment:
So this seems to be the class for the `DocumentTask`. This task already exists as a skeleton. 

I would be of the opinion that it should live there and extend/flesh out the existing one for the following reasons:
- Task can be based on a `BaseTask` ABC
- Tasks should live in `task` so we don't have to wonder where to find them
- The skeleton of the task factory in `main.py` https://github.com/bitpicky/dbt-sugar/blob/main/dbt_sugar/core/main.py already knows how to pick it and call it.

Do you buy that?

Reply:
Yes, sounds good! will move it there!